Sources/MapboxMaps/Foundation/Events/EventsManager.swift

```diff
@@ -28,6 +35,7 @@ internal final class EventsManager {
     }
 
     private let mmeEventsManager: MMEEventsManager
```

Comment:
Now when `mmeEventsManager` is not used here to send any events, could it be removed altogether? Or is it so that it sends some events by itself?